modules/ROOT/pages/api-led-design.adoc

```diff
@@ -0,0 +1,159 @@
+                 message: string
+               example: |
+                 {
+                   "message" : "Greeting not found"
+                 }
+----
+<1> A single HTTP request, GET
+<2> A single `dataType`, `greeting`, with a single property, `todays-greeting` (today’s greeting message). 
+<3> HTTP response
+<4> Error message and error handling
+
+== Step 3: Test Your API Specification
+Now that you’ve created a Hello World specification, test it by sending a request.
+The mocking service handles the creation of a functioning endpoint from your API specification
+and provides a simple UI for managing authentication, request headers, and response headers.
+
+To test your Hello World specification:
+
+. link:https://anypoint.mulesoft.com/designcenter/#/projects["Open the Hello World API specification that you created in Step 2.^" role="button-primary"]
+. Click the X next to *Mocking service* to enable the mocking service. If you see a checkmark instead of an X, the mocking service is already enabled.
+. Inspect the *API endpoints* that you defined. HTTP requests appear in green boxes.
+. Click *GET* to display the GET endpoint.
+. Review the *Code examples* for each protocol.
+. Review the *Security* method. *Basic Authentication* is the only option.
+. Click *Try it*.
+. Enter values in the Authorization and Parameters tab: 
+  .. In the Authorization tab, enter the *User name* and *Password* needed to access this endpoint. In this example, you can enter any string.
+  .. In the Parameters tab, enter a value for *todays-greeting*: `Hello World`. 
+. Click *Send* to send your request to the temporary request URL that the mocking service creates from your specification. If the request is successful, you’ll see `*HTTP message 200 OK*`. 
+. Click *Details* to examine the response headers and request headers in the mocking service to help diagnose issues or understand the behavior of your API specification.
+. When you have finished testing, click the mocking service checkmark to exit.
+
+== Step 4. Publish Your API Specification
+After you have tested your API, publish it to your private Exchange so others in your organization can reuse your work.
+
+. link:https://anypoint.mulesoft.com/designcenter/#/projects["Open the Hello World API specification.^" role="button-primary"]
+. Click Publish.
+. Click Publish to Exchange. 
+
+Now anyone in your organization can see the Hello-World API specification and reuse it.
+
+== What’s Next
+
+Now that you have designed an API and specified it so that anyone can use it, create a Mule app that contains the implementation and interface for the API, using Anypoint Studio.
+
+== Developer Deep Dives
+
+If you're curious about the details, dive right in.
+
+=== Deep Dive: Exchange
+
+You can publish assets to the public Exchange, your internal exchange, or your public developer portal.
+
+* In addition to the public Exchange, you can review your own organization’s internal offerings.
```

Comment:
```suggestion
* In addition to the public Exchange, you can review your own organization’s internal offerings:
```

Reply:
This isn't a link, it's a button. You need to look at the beta doc while you review these. Recommend two-browser setup (wide screen at work allows for this).

Reply:
As a button, isn't this a form of illustration? I would put a colon at the end of text that introduced a graphic. 
But I get that this is not a graphic so much as a control, so...active, not an illo.

Reply:
No. It's a button that, if you click it, takes you somewhere. We're going to go with "Take me there" as the text for all of them.

Reply:
FYI, button text is sentence case: http://ux-v3.mulesoft.com/#/playground/Button